ts/IterableInbox.tsx

```diff
@@ -81,13 +95,12 @@ const IterableInbox = ({
             <Text style={styles.headline}>
                {customizations.navTitle ? customizations.navTitle : defaultInboxTitle}
             </Text>
-            { messages.length ?
+            { rowViewModels.length ?
                <IterableInboxMessageList 
-                  messages={messages}
-                  messageListItemLayout={messageListItemLayout}
+                  rowViewModels={rowViewModels}
```

Comment:
It seems like `messageListItemLayout` got removed from here? I'm not sure why, but it's also showing as an error.

Reply:
Addressed.